downloader.py

```diff
@@ -10,6 +10,24 @@
 __log__ = logging.getLogger(__name__)
 
 
+def download_media(client, msg, target_id):
+    media = msg.media
+    os.makedirs('usermedia', exist_ok=True)
+    file_name_prefix = '%d-%d-' % (target_id, msg.id)
```

Comment:
Can you change this for `'{}-{}-'.format(...)`? To be more consistent.

Reply:
Agreed, it's nice to have a single style across the project